tools/ci/run_tc.py

```diff
@@ -146,25 +146,20 @@ def start_dbus():
     # https://source.chromium.org/chromium/chromium/src/+/main:content/app/content_main.cc;l=220;drc=0bcc023b8cdbc073aa5c48db373810db3f765c87.
     os.environ["DBUS_SESSION_BUS_ADDRESS"] = "autolaunch:"
 
-
 def install_chrome(channel):
-    if channel in ("experimental", "dev"):
-        deb_archive = "google-chrome-unstable_current_amd64.deb"
-    elif channel == "beta":
-        deb_archive = "google-chrome-beta_current_amd64.deb"
-    elif channel == "stable":
-        deb_archive = "google-chrome-stable_current_amd64.deb"
-    else:
-        raise ValueError("Unrecognized release channel: %s" % channel)
-
-    dest = os.path.join("/tmp", deb_archive)
-    deb_url = "https://dl.google.com/linux/direct/%s" % deb_archive
-    with open(dest, "wb") as f:
-        get_download_to_descriptor(f, deb_url)
-
-    run(["sudo", "apt-get", "-qqy", "update"])
-    run(["sudo", "gdebi", "-qn", "/tmp/%s" % deb_archive])
-
+    # Interpret experimental channels as Canary.
+    if channel in ("experimental", "nightly", "canary"):
+        channel = "canary"
+    run(["sh", "-c", "./wpt install --channel canary chrome browser"])
+
+    # Chrome for Testing needs additional files in the download directory to run all features.
+    file_names = os.listdir(os.path.join(os.getcwd(),
+                                         f"_venv3/browsers/{channel}/chrome-linux64"))
+    for file_name in file_names:
+        path = os.path.join(os.getcwd(),
+                            f"_venv3/browsers/{channel}/chrome-linux64",
+                            file_name)
+        run(["sudo", "mv", path, "/usr/bin"])
```

Comment:
Is this actually required or is it just a simplification so that `which` will pick things up later? In the latter case I'd strongly prefer not to do this.

Reply:
As far as I could tell with the integration tests (which this setup is for), The venv folder is removed or rebuilt at some point after this step, and the browser binary cannot be found. `/usr/bin` is where the previous Chrome Dev binary was installed to, and where the Firefox binary exists as well.